fe/fe-core/src/main/antlr4/org/apache/doris/nereids/PLParser.g4

```diff
@@ -324,11 +325,15 @@ package_body_item :
       declare_stmt_item
     | create_function_stmt
     | create_procedure_stmt
+    | drop_procedure_stmt
```

Comment:
In /hive/hplsql/src/main/antlr4/org/apache/hive/hplsql/Hplsql.g4
![image](https://github.com/apache/doris/assets/13197424/26b2b51c-d4b9-4341-91f7-5554da0f24bf)

no `drop_procedure_stmt`, but I don't know why.

Reply:
I checked hplsql.g4, I think it's implemented as drop_stmt. do you think we should follow same ?  I had followed createprocedure method.  

what is your WeChat id? can I add you there?

Reply:
sure, my WeChat id is CatCatBlueBlue.